# This message will be used monitor information specific to the robolink
# joint_name is a string representing the name of the joint
# joint_num is the number of the joint
# joint_angle is the angle of the joint in degrees
# joint_velocity is the velocity of the joint in degrees per second


geometry_msgs/Pose current_position
int32[] joint_angles
int32[] joint_velocities
time stamp
